### PR TITLE
[UOT-121160] Create GC ML API Classification endpoint to support JBOOK text classification model

### PR DESCRIPTION
**Pull request by mishoe**
**Jira Ticket:** https://jira.di2e.net/browse/UOT-121160
**Description:** This PR updates the GC ML API by adding a new endpoint to perform JBOOK record classification. The first version of the transformer based classifier model has been staged in s3 at `s3://advana-data-zone/bronze/gamechanger/models/classification/v1/distilbert_classifier_20211112.tar.gz`

### CODE REVIEW

**Comment by mishoe** [comment]
Until confirmation that the `s3` pathing for the model is correct (being checked  @ds-ramon), keeping this PR in draft

File: gamechangerml/setup_env.sh [hunk 1]
```diff
@@ -22,7 +22,8 @@ function setup_prod() {
     export S3_QEXP_PATH="${S3_QEXP_PATH:-s3://advana-data-zone/bronze/gamechanger/models/qexp_model/v3/qexp_20201217.tar.gz}"
     export S3_TOPICS_PATH="${S3_TOPICS_PATH:-s3://advana-data-zone/bronze/gamechanger/models/topic_model/v1/20210208.tar.gz}"
     export S3_ML_DATA_PATH="${S3_ML_DATA_PATH:-s3://advana-data-zone/bronze/gamechanger/ml-data/v1/data_20211018.tar.gz}"
-    
+    export S3_CLASSIFY_JBOOK_MODEL_PATH="${S3_CLASSIFY_JBOOK_MODEL_PATH:-s3://advana-data-zone/bronze/gamechanger/models/classification/v1/distilbert_classifier_20211112.tar.gz}"
+
     export DOWNLOAD_DEP="${DOWNLOAD_DEP:-true}"
     export ES_HOST="${ES_HOST}:-}"
```
Comment (mishoe) [inline gamechangerml/setup_env.sh:28]: @rha930 not sure if you noticed this yourself, but `export ES_HOST="${ES_HOST}:-}"` is throwing a value error
```
gc-ml-gpu             |   File "/usr/lib64/python3.8/urllib/parse.py", line 175, in port
gc-ml-gpu             |     raise ValueError(message) from None
gc-ml-gpu             | ValueError: Port could not be cast to integer value as '-}'
```

**Comment by mishoe** [comment]
@rha930 and @cWren0110 this PR is ready for review 👍

**Comment by cWren0110** [comment]
How is this PR coming.  Does it need additional review or testing? @mishoe

**Comment by cWren0110** [comment]
is this branch OBE or does more need to be done?  @mishoe

**Comment by mishoe** [comment]
Closing for now (will re-open if we decide to use this model/endpoint in production for JBOOK)

**Pull request closed without merging**